### ISSUE TITLE
docker-compose running as root

### ISSUE THREAD

**Issue by jkonrath** [pirate/ArchiveBox#437]
I'm trying out docker-compose and it's running as root, which gives me an error.

I.e.
```
$ echo "https://example.com" | docker-compose exec -T archivebox archivebox add
[i] [2020-08-11 20:00:18] ArchiveBox v0.4.11: archivebox add < /dev/stdin
    > /data

[!] ArchiveBox should never be run as root!
    For more information, see the security overview documentation:
        https://github.com/pirate/ArchiveBox/wiki/Security-Overview#do-not-run-as-root
```

Adding `ALLOW_ROOT=True` to my `docker-compose.yml` didn't do anything.

Also, a few nitpicky doc things on https://github.com/pirate/ArchiveBox/wiki/Docker:

* it still says `exec -T archivebox /bin/archive` - should that last bit be `archivebox add` ?
* It says `http://127.0.0.1:8080` - should be `8000`

**Comment by coisnepe** [comment]
Running into this issue as well. Did you make any progress?

**Comment by jkonrath** [comment]
Haven't had time to try this, but according to https://dev.to/acro5piano/specifying-user-and-group-in-docker-i2e this might be a workaround:

1. Add this to `docker-compose.yml`:  `user: "${UID}:${GID}"`
2. Start like this:
```
export UID=${UID}
export GID=${GID}
docker-compose up
```

**Comment by cdvv7788** [comment]
What os are you using? It is happening to me on macos...just want to confirm it happens in linux too.

**Comment by jkonrath** [comment]
I'm also macos. (10.15.4)

**Comment by cdvv7788** [comment]
Try running `echo "https://example.com" | docker-compose run -T archivebox add`

**Comment by jkonrath** [comment]
That works!
What I ended up doing was changing the docker-compose.yml as you said for a local build, then built locally and it was fine. No need for setting the user. (I actually tried setting the user and that fails.) 

(One minor problem: https://example.com doesn't actually exist. :) http works. That tripped me up for a minute...)

I can help with docs when I get some time, maybe this weekend. I'll take a pass and send a pull request when I get a chance.
